src/btree/bt_vrfy.c

```diff
@@ -229,21 +230,24 @@ __verify_row_leaf_page_hs(WT_SESSION_IMPL *session, WT_REF *ref, WT_VSTUFF *vs)
 
 /*
  * __verify_key_hs --
- *     Verify a key against the history store given a cell unpack holding key values.
+ *     Verify a key against the history store. Unpack will be used to verify the data continuity
```

Comment:
This comment is a bit misleading. You're only using unpack on to check one of the timestamp ranges. The remaining ones are using the time pairs embedded in the history store key to check data continuity.